k8s_log_watcher/agents/appdynamics.py

```diff
@@ -9,6 +9,6 @@ def get_template_path():
 
 def get_template_vars(pod_name, pod_labels):
     return {
-        'app_name': pod_labels.get('app_name'),
-        'app_tier': pod_labels.get('app_tier'),
+        'app_name': pod_labels.get('appdynamics_app'),
+        'app_tier': pod_labels.get('appdynamics_tier'),
```

Comment:
As I understood, we won't use namespaces, and use ``app`` & ``version`` directly (as prefixes are optional). I am also kind of wary of Z specific key words slipping into opensource projects. Is there a label prefix we use elsewhere?